### PR TITLE
data: saifrahmn

### PR DESCRIPTION
**Pull request by saifrahmn**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5474"></a>

### CODE REVIEW

File: public/data/saifrahmn.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "name": "Work with me",
+      "url": "https://www.linkedin.com/in/saif-rahman-a62686201",
+      "icon": "LinkedIn"
+    },
+      {
+         "name": "Visit my Personal Website",
+         "url": "https://clomen.web.app/",
+         "icon": "Globe"
+       }
+
```
Comment (schmelto) [inline public/data/saifrahmn.json:43]:
```suggestion
```

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey @saifrahmn please move your file from `public/data` to `data` folder

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
As previously requested, please allow the docs on updating your fork and your profile. We have a step by step guide and a video https://linkfree.eddiehub.io/docs/how-to-guides/editing

File: data/saifrahmn.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
+    "avatar": "https://github.com/saifrahmn.png",
```
Comment (eddiejaoude) [inline data/saifrahmn.json:5]: Avatar is not required, please read the docs for the fields https://linkfree.eddiehub.io/docs/quickstart

File: data/saifrahmn.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "links": [
+      {
+        "name": "For Collaboration",
+        "url": "https://github.com/saifrahmn",
+        "icon": "github"
```
Comment (eddiejaoude) [inline data/saifrahmn.json:10]: these are not validate icons, please read the docs https://linkfree.eddiehub.io/docs/quickstart

File: data/saifrahmn.json [hunk 4]
```diff
@@ -1,43 +1,42 @@
 {
-    "name": "Saif Rahman",
-    "type": "personal",
-    "bio": "Engineer-Student-Enthusiast",
-    "avatar": "https://github.com/saifrahmn.png",
+  "name": "Sara Jaoude",
```
Comment (khushalarora) [inline data/saifrahmn.json:2]: Please replace this with your own name "Saif Rahman"

File: data/saifrahmn.json [hunk 5]
```diff
@@ -1,43 +1,42 @@
...
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```
Comment (khushalarora) [inline data/saifrahmn.json:4]: Replace this with your own bio

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @saifrahmn, you have not used your own name and Bio. Please correct them. :+1:

File: data/saifrahmn.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "Sara Jaoude",
+  "name": "Saif Rahman",
   "type": "personal",
-  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "bio": "Student - Engineer - Snthusiast",
```
Comment (khushalarora) [inline data/saifrahmn.json:4]: hey @saifrahmn, Spelling of **'Enthusiast'** is not correct here :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Please remove your duplicate profile with the incorrect username

**Comment by eddiejaoude** [comment]
> I can not find my files in the repository, even when a search in saifrahmn:patch-1
> [these is the folder i am searching in](https://github.com/saifrahmn/LinkFree/tree/patch-1/data)

I share a link for you already in my review https://github.com/EddieHubCommunity/LinkFree/pull/5474#pullrequestreview-1343299189

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by eddiejaoude** [comment]
After I merge this please update anywhere you use your LinkFree url, as now it is using your correct GitHub username, your LinkFree url will change

**Comment by saifrahmn** [comment]
Thanks @eddiejaoude

**Pull request merged**